Azul

PJF98 opened the pull request:
Adding the Azul game. Thought it was about time I cleaned up the code a bit and pushed it back to the original repo.

I only coded the game for 2 players although the game can be played with more but I wasn't interested in that.

Let me know if/what code changes you want for it to me merged. I notice you use tabs instead of spaces and also my linter is quite specific about certain things.

Also if you don't want new games added let me know and I'll keep them to my repo.

Thanks a lot, Peter

cestpasphoto commented:
Hi @PJF98 ,

I don't have time to look in the details of your code. I see that you haven't modified any common file which is good (except GameSwitcher but that is needed for sure).
Do you want to modify the root README file as well to describe the game and show an example? Not mandatory at all, just asking.

I won't have time to review, so I'll trust you and merge once we decide the above.

PJF98 commented:
Sure, very happy to record an example. What program did you use to make your screen recordings?

cestpasphoto commented:
I tried several methods but the one with the best results was the regular video screenshot, then converted to low-res gif. FYI I use Linux / Debian, maybe there's better tools on Windows or Mac.

PJF98 commented:
I'm using Linux Mint so that might be the best approach for me too

PJF98 commented:
I've added a gif and updated the README file. The gif didn't come out as nice as your gifs but I think it's passable. It seems my Linux version didn't come with an inbuilt screen record so I recorded with SimpleScreenRecorder and then converted to a gif with ffmpeg.

cestpasphoto commented:
Nice job and thank you very much!
